modules/network-observability-filtering-ebpf-rule.adoc

```diff
@@ -29,46 +27,29 @@ spec:
   agent:
     type: eBPF
     ebpf:
+      privileged: true <1>
+      features: <2>
+        - PacketDrop
+        - FlowRTT
       flowFilter:
-        action: Accept  <1>
-        cidr: 172.210.150.1/24 <2>
-        protocol: SCTP
-        direction: Ingress
-        destPortRange: 80-100
-        peerIP: 10.10.10.10
-        enable: true    <3>
+        enable: true <3>
```

Comment:
Customer can view Egress traffic and PacketDrop(blog) for Infra

enable: true <3>
        rules:
         - action: Accept  <1>
        cidr: 0.0.0.0/0 <2>
        protocol: TCP
        direction: Egress
        sourcePort: 100
        peerIP: 192.168.127.12 <3>
        enable: true    <4>
          - action: Accept
            cidr: 172.30.0.0/16
            pktDrops: true <6>